### PR TITLE
Orbit chain ownership

### PR DESCRIPTION
**Pull request by anegg0**
- Added 'upgrade executor' to the glossary
- Fixed corresponding `data-quicklookup-from` 
- Improved some phrasing

### CODE REVIEW

File: arbitrum-docs/launch-orbit-chain/concepts/chain-ownership.md [hunk 1]
```diff
@@ -19,15 +19,15 @@ import PublicPreviewBannerPartial from '../partials/_orbit-public-preview-banner
 
 ### Architecture
 
-Chain ownership affordance is handled via [**Upgrade Executor**](https://github.com/OffchainLabs/upgrade-executor) contracts.
+Chain ownership affordance is handled via <a data-quicklook-from='upgrade-executor'>Upgrade Executors</a> contracts.
 
-Each Orbit chain is deployed with two Upgrade Executors — one on the Orbit chain itself, and one on its <a data-quicklook-from='parent-chain'>parent chain.</a> At deployment, the chain's critical affordances are given to the Upgrade Executor contracts.
+Each Orbit chain is deployed with two upgrade executors. — one on the Orbit chain itself, and one on its <a data-quicklook-from='parent-chain'>parent chain.</a> At deployment, the chain's critical affordances are given to the Upgrade Executor contracts.
```
Comment (symbolpunk) [inline arbitrum-docs/launch-orbit-chain/concepts/chain-ownership.md:24]:
```suggestion
Each Orbit chain is deployed with two upgrade executors — one on the Orbit chain itself, and one on its <a data-quicklook-from='parent-chain'>parent chain.</a> At deployment, the chain's critical affordances are given to the Upgrade Executor contracts.
```

File: arbitrum-docs/launch-orbit-chain/concepts/chain-ownership.md [hunk 2]
```diff
@@ -19,15 +19,15 @@ import PublicPreviewBannerPartial from '../partials/_orbit-public-preview-banner
...
+Each Orbit chain is deployed with two upgrade executors — one on the Orbit chain itself, and one on its <a data-quicklook-from='parent-chain'>parent chain.</a> At deployment, the chain's critical affordances are given to the Upgrade Executor contracts.
 
 Some examples:
 
 - The parent chain's core protocol contracts are upgradeable proxies that are controlled by a proxy admin; the proxy admin is owned by the Upgrade Executor on the parent chain.
 - The core Rollup contract's admin role is given to the Upgrade Executor on the parent chain.
-- The affordance to call setters on the ArbOwner procompile — which allows for setting system gas parameters and scheduling ArbOS upgrades (among other things) — is given to the Upgrade Executor on the Orbit chain.
+- The affordance to call setters on the ArbOwner precompile — which allows for setting system gas parameters and scheduling   <a data-quicklook-from='arbos'>ArbOS.</a> upgrades (among other things) — is given to the Upgrade Executor on the Orbit chain.
```
Comment (symbolpunk) [inline arbitrum-docs/launch-orbit-chain/concepts/chain-ownership.md:30]:
```suggestion
- The affordance to call setters on the ArbOwner precompile — which allows for setting system gas parameters and scheduling  <a data-quicklook-from='arbos'>ArbOS</a> upgrades (among other things) — is given to the Upgrade Executor on the Orbit chain.
```

**Comment by symbolpunk** [comment]
In review (walking through the `update:all` / `update.ts` workflow)

**Comment by anegg0** [comment]
Will have to run the glossary script locally and create another PR.

**Pull request closed without merging**